geometry_msgs/PoseStamped p
string arm
---
